templates/templates/blockified/archive-product.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import {
+	createBlock,
+	createBlocksFromInnerBlocksTemplate,
+} from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import {
+	INNER_BLOCKS_TEMPLATE as productsInnerBlocksTemplate,
+	QUERY_DEFAULT_ATTRIBUTES as productsQueryDefaultAttributes,
+} from '../../../assets/js/blocks/product-query/constants';
+import { VARIATION_NAME as productsVariationName } from '../../../assets/js/blocks/product-query/variations/product-query';
+
+const productArchiveClassName = 'woo-blocks-product-archive-template';
+const commonAttributes = {
+	className: productArchiveClassName,
+};
```

Comment:
Heads-up that [we usually use the `wc-blocks-` prefix](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/contributors/contributing/coding-guidelines.md#prefixing) for class names. Having said that, do we really need to add a custom class name here?